### ISSUE TITLE
[not validated] History and dynamic adding don't work together

### ISSUE THREAD

**Issue by GenaBitu** [rejas/imagelightbox#167]
I have a very strong feeling that history and dynamic adding won't work together due to the way history is implemented. Oh, well...

Will look into it more.

**Comment by GenaBitu** [comment]
Yep, popback is broken. Who knows what reloading an open image will do, since it may not have been added to the lightbox yet.

**Comment by rejas** [comment]
Shows how many people so far have used those functonalities together :-D
